tests/components/devolo_home_network/test_sensor.py

```diff
@@ -89,7 +90,7 @@ async def test_update_neighboring_wifi_networks(hass: HomeAssistant):
         assert state.state == "1"
 
         er = entity_registry.async_get(hass)
-        assert er.async_get(state_key).entity_category == ENTITY_CATEGORY_DIAGNOSTIC
+        assert er.async_get(state_key).entity_category == EntityCategory.DIAGNOSTIC
```

Comment:
For entity_category, you can use `is`. Can you please open a separate PR to check if the corresponding type hint should be adjusted for `EntityCategory` ?
https://github.com/home-assistant/core/blob/f55668ff40c4c0a53411e53d6334e51d64990867/homeassistant/helpers/entity_registry.py#L108